public/Day 25/index.html

```
@@ -1,68 +1,63 @@
-                <button id="nextBtn">Next Quote</button>
-                <button id="changeBtn">Change Category</button>
-            </div>
-
+<div class="container">
+
+<h1>✨ Theme Quote Hub</h1>
+
+<div class="section">
+    <h2>➕ Add Quote</h2>
+
+    <input type="text" id="newQuote" placeholder="Enter quote">
```

HTMLHint (1.9.1)

[warning] The empty tag : [ input ] must be self closed.

(tag-self-close)